backend/orchestrators/LoadBalancer.py

```diff
@@ -0,0 +1,53 @@
+import random
+import os
+import json
+from dotenv import load_dotenv
+
+load_dotenv()
+class OpenAIContext():
+    def __init__(self, resource: str, model: str, endpoint_url: str, key: str, weight: float, version: str):
+        # these are the values required to create each endpoint context
+        self.resource = resource
+        self.model = model
+        self.endpoint_url = endpoint_url
+        self.key = key
+        self.weight = weight
+        self.version = version
+
+    def to_dict(self):
+            return {
```

Comment:
Extra indent level on the return statement. Might be problematic in some Python environments.